src/alto-ui/Actions/__tests__/Actions.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Actions from '../Actions';
+
+import Dropdown from '../../Dropdown';
+import Group from '../../Group';
+
+jest.mock('../../Dropdown', () => {
+  return props => {
+    return <button className="Dropdown" onClick={() => props.onClick()} />;
+  };
+});
+
+jest.mock('../../Group', () => {
+  return  () => {
+    return <div className="Group" />;
+  };
+});
+
+describe('<Actions />', () => {
+    const defaultProps = {
+      className: 'classTest',
+      id: 'id',
+      items: [
+        {
+          icon: null,
+          key: '1',
+          onClick: () => [],
+          title: 'Edit',
+        },
+        {
+          icon: null,
+          key: '2',
+          onClick: () => [],
+          title: 'Edit',
+        },
+      ],
+      max: 1
+    };
+
+  const getWrapper = ({ ...props }) => shallow(<Actions {...defaultProps} {...props} />);
+
+  it('should not render if no items are provided', () => {
+    expect(getWrapper({items: undefined}).type()).toBe(null);
```

Comment:
```suggestion
    expect(getWrapper({items: undefined}).type()).toBeNull();
```

Reply:
fixed in "fixes after code review" commit